### ISSUE TITLE
Spatiotemporal slowfast head loss question

### ISSUE THREAD

**Issue by zimka** [open-mmlab/mmaction2#508]
The configs from configs/detection/ava/slowfast*.py use BboxHead, whose loss is defined [here](https://github.com/open-mmlab/mmaction2/blob/master/mmaction/models/heads/bbox_head.py#L160)
It looks like classification loss does not take into account class 0 (`label = label[:, 1:]`), which as far as I understand means background/catch-all class (following Fast R-CNN implementation). If my understanding is correct, False Positive detections are not taken into account when loss is computed, which looks kinda strange.
Is my assumption correct, or the meaning of this line is different? If it is correct, could you provide any hints why is it implemented in this way?
In the fast rcnn paper localization loss is not applied to background predictions for obvious reasons, but classification loss is applied for all logits including background.

**Comment by kennymckormick** [comment]
Currently, the training of Spatio-temporal action detection doesn't take background class into consideration. We only use positive examples[IoU > 0.9] in training, and use proposals with high detection scores in testing. In a word, all proposals as positive in our current implementation. 
The class 0 is reserved for further usage (to identify whether a proposal is positive or negative). But this feature is not implemented now.
